acct-user/snapserver/snapserver-0.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="A user for the Snapcast server component"
+ACCT_USER_ID=114
+ACCT_USER_HOME=/var/lib/snapserver
```

Comment:
And here.

Reply:
snapserver creates the `server.json` file in its home dir, so this needs to be set.